test/testUtils/ufeUtils.py

```diff
@@ -67,6 +67,18 @@ def createUfeSceneItem(dagPath, sdfPath=None):
     ufeItem = ufe.Hierarchy.createItem(ufePath)
     return ufeItem
 
+def createItem(ufePathOrPathString):
```

Comment:
Utilities to simplify UFE scene item and hierarchy interface creation.  Should have done this a while ago...  Should be deployed to other tests as we maintain them.

Reply:
Given that I never remember the incantations, it will help me!